internal/pkg/scorecard/olm_tests.go

```diff
@@ -237,10 +235,11 @@ func (t *CRDsHaveValidationTest) Run(ctx context.Context) *TestResult {
 // Run - implements Test interface
 func (t *CRDsHaveResourcesTest) Run(ctx context.Context) *TestResult {
```

Comment:
I missed this pointer return yesterday. Can we return a `TestResult` here?

Reply:
Yeah, we can change that to return the struct instead of a struct pointer. It's outside of the scope of changes in this PR though, so I'll do that in a separate PR.